crates/rpc/fixtures/0.6.0/class_hash/at_block.json

```diff
@@ -0,0 +1 @@
+"0x636c61737320312068617368"
```

Comment:
I agree with GitHub (and other tools) that text files should end with a newline... :-)

Reply:
Doesn't really affect the comparisons which is what we care of imho. Thin of it like a string (it is being imported as such). That newline is irrelevant.